packages/@okta/vuepress-site/docs/guides/add-an-external-idp/create-an-app-at-idp/openidconnect/appatidp.md

```diff
@@ -1,7 +1,7 @@
-Use the Identity Provider's documentation to create a client application. For example, if you want to use PayPal as your OpenID Connect Identity Provider, click **Create App** on [this page](https://developer.paypal.com/developer/applications/) to create a PayPal client application.
+Use the Identity Provider's documentation to create a client application, for example, when you want to use your existing Identity Provider as your OpenID Connect Identity Provider.```
```

Comment:
....when you want to use your existing Identity Provider as your OpenID Connect provider.